packages/testcontainers/src/containers/DeFiDContainer.ts

```diff
@@ -35,7 +35,7 @@ export abstract class DeFiDContainer extends DockerContainer {
     if (process?.env?.DEFICHAIN_DOCKER_IMAGE !== undefined) {
       return process.env.DEFICHAIN_DOCKER_IMAGE
     }
-    return 'defi/defichain:HEAD-49fba65ce'
+    return 'defi/defichain:master-1dff273bf'
```

Comment:
For this change, do we need to update to the latest image? 

Or will it also work with the existing image?

Reply:
I believe the image is required, the RPC was added in https://github.com/DeFiCh/ain/commit/e8ca8a65ab980fd2ed7ba978bbc7400bb5ac0655.

Reply:
Oh, I guess we should merge after v3 PR are merged since there is breaking changes with that image.